vector/src/main/java/im/vector/app/features/notifications/NotificationUtils.kt

```diff
@@ -917,7 +916,7 @@ class NotificationUtils @Inject constructor(private val context: Context,
         }
 
         // We cannot use NotificationManagerCompat here.
-        val setting = context.getSystemService<NotificationManager>()!!.currentInterruptionFilter
+        val setting = context.getSystemService(NotificationManager::class.java)!!.currentInterruptionFilter
```

Comment:
Is this change necessary?

Reply:
Not necessary but fixes some linting for me locally should i revert?

Reply:
What is the lint issue you are seeing? I do not see it, nor the CI.

Reply:
Ok i will revert this.